src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -456,7 +456,7 @@ private IEnumerable<(ManifestId, ManifestVersion)> ParseRollbackDefinitionFile(s
         private bool BackgroundUpdatesAreDisabled() =>
             bool.TryParse(_getEnvironmentVariable(EnvironmentVariableNames.WORKLOAD_UPDATE_NOTIFY_DISABLE), out var disableEnvVar) && disableEnvVar;
 
-        private string GetAdvertisingManifestSentinalPath() => Path.Combine(_userProfileDir, ".workloadAdvertisingManifestSentinal");
+        private string GetAdvertisingManifestSentinalPath(SdkFeatureBand featureBand) => Path.Combine(_userProfileDir, ".workloadAdvertisingManifestSentinal" + featureBand.ToString());
```

Comment:
You can probably just use interpolation and avoid calling ToString explicitly, `$".workloadAvertisingManifestSentinel{featureband}"` Also, should there be a separator between the filename and version like "." or "-"?